src/components/layout.js

```diff
@@ -7,12 +7,22 @@ import styles from "./layout.module.scss"
 import Navbar from "./navbar"
 import Logo from "./logo"
 
-export default class Layout extends React.PureComponent {
-  state = {
-    value: "",
-  }
+const renderNavItems = items => (
+  <ul>
+    {items.map(({ name, endpoint, anchor = false }, index) => (
+      <li key={`footer nav item ${name}`} className={styles.footerNavItem}>
+        {anchor ? (
+          <a href={endpoint}>{name}</a>
```

Comment:
Does Link not support anchors?

Reply:
Links can only point to endpoints inside the Gatsby app, you have to use anchor tags to point to external sites.